f3-sidecar/ffi_gen.go

```diff
@@ -0,0 +1,210 @@
+package main
```

Comment:
Include this file so that the Go mod builds out of box without codegen from Rust build